integrations/ethereum-blobs/utils/utils.ts

```diff
@@ -0,0 +1,36 @@
+import crypto from "crypto";
+import axios from "axios";
+
+export async function getTransactionByHash(rpc: string, hash: string) {
+  const data = {
+    method: 'eth_getTransactionByHash',
+    params: [hash],
+    id: 1,
+    jsonrpc: '2.0'
+  };
+
+  try {
+    const response = await axios.post(rpc, data, {
+      headers: {
+        'Content-Type': 'application/json'
+      }
+    });
+    return response.data.result;
+  } catch (error) {
+    console.error('Error:', error);
+  }
+}
+
```

Comment:
can you add a comment what this method is doing? It's very hard to understand what this method is for